bluetooth: fast_pair: Set Oberon as default crypto backend (#10949)
nrfconnect/sdk-nrf: doc/nrf/libraries/bluetooth_services/services/fast_pair.rst

* :kconfig:option:`CONFIG_BT_FAST_PAIR_STORAGE_ACCOUNT_KEY_MAX` - The option configures maximum number of stored Account Keys.
* :kconfig:option:`CONFIG_BT_FAST_PAIR_CRYPTO_TINYCRYPT`, :kconfig:option:`CONFIG_BT_FAST_PAIR_CRYPTO_MBEDTLS`, and :kconfig:option:`CONFIG_BT_FAST_PAIR_CRYPTO_OBERON` - These options are used to select the cryptographic backend for Fast Pair.
  The Oberon backend is used by default.

>   The Mbed TLS backend is using Mbed TLS crypto APIs which is considered a legacy API.

Reviewer: ```suggestion
  The Mbed TLS backend uses the Mbed TLS crypto APIs, which are now considered legacy APIs.
```

Progressive tense to present (is using -> uses), subject verb agreement (APIs vs API, is -> are).
